scss/responsive.scss

```diff
@@ -0,0 +1,62 @@
+@import 'variables';
+
+@media screen and (max-width: 700px){
+
+  .navbar-text {
+    visibility: hidden;
+  }
+
+  .card-container {
+    display: grid;
+    grid-template-columns: repeat(1, max-content);
+    justify-content: center;
+    align-items: center;
+    max-width: 1200px;
+    width: 100%;
+    margin: 50px auto;
+  }
+
+  .background-card{
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 1px solid $green;
+    flex-direction: column;
+    font-size: 14px;
+    height: 370px;
+    width: 220px;
+    position: relative;
+    margin-left: 20px;
+    z-index: 1;
+  }
+
+  .main-tech {
+    border-botom: 1px solid $green;
+    border-right: 0;
+    grid-row: 1/3;
+    height: 100%;
+    justify-content: center;
+  }
+
+  .title {
+
+    padding: 195px 30px 30px 150px;
+    margin-left: -20px;
+    font-size: $font-big;
+    position: relative;
+    z-index: 1;
```

Comment:
With `align-self: center;` I think it's enough :smile:

Reply:
Using <code>align-self: center;</code> forces a line break in the title and the text lines overlap with each other.

![overlap_1](https://user-images.githubusercontent.com/9786975/34307631-ce22c81c-e727-11e7-89e3-edbe7ef8a959.png)

Would it be ok just to leave the <code>margin-left: -20px;</code> instead? (and deleting the other unnecesary attributes, obviously)